app/models/jellyfish_demo/project/generic.rb

```diff
@@ -0,0 +1,10 @@
+module JellyfishDemo
```

Comment:
This file should be deleted.

- `Project` cannot be a module, it is already a class that inherits from `ActiveRecord::Base` loading this file before Core would break Core.
- Projects do not support STI, so `Project::Generic` cannot be used.
- Projects cannot be loaded this way.